ci-operator/config/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master__v4.10.yaml

```diff
@@ -20,12 +20,18 @@ images:
       - destination_dir: .
         source_path: /usr/bin/oc
   to: oc-bin-image
+- context_dir: .
+  from: src
+  to: gitops-backend-operator
 operator:
   bundles:
   - as: gitops-operator-bundle
     base_index: operator-index
     context_dir: bundle/
     dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/redhat-developer/gitops-backend-operator:v0.0.3
```

Comment:
This looks fishy. How is it going to work when the image tag is changing?

Reply:
We replace this particular image with the image built by CI. That way we get the operator bundle with latest changes

Reply:
Are we removing all the tags?

Reply:
The motivation behind removing the tag initially was that, if a user tries to modify the tag in GitHub GitOps repository in a pull request, the CI would break and somebody has to update the tag in GitHub Release repository as well and they might not be aware of it. But even if we remove the tag here, when someone raises a pull request in GitHub GitOps repository with bundle specific changes, the tag would automatically be added and would cause the same problem as mentioned before.

cc: @iam-veeramalla